loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java

```diff
@@ -58,6 +66,10 @@
     public static final boolean DEFAULT_SIMUL_SHUNT = false;
     public static final boolean DEFAULT_READ_SLACK_BUS = false;
     public static final boolean DEFAULT_WRITE_SLACK_BUS = false;
+    public static final boolean DEFAULT_VOLTAGE_REMOTE_CONTROL = false;
+    public static final boolean DEFAULT_DC = false;
+    public static final boolean DEFAULT_DISTRIBUTED_SLACK = false;
```

Comment:
I think the default value of distribution slack should be true here. @geofjamg ?